Introduce count_EQ and deprecate the implicit count filter. (#5636)
neo4j/graphql: .changeset/shaggy-swans-collect.md

---
"@neo4j/graphql": minor
---

> Added aggregation filter `count_EQ` filters: `{ count_EQ: 10 }`, this is an alternative version of the deprecated version `{ count: 10 }`. 

Reviewer: ```suggestion
Added aggregation filter `count_EQ` filters: `{ count_EQ: 10 }`, this is the replacement for the deprecated version `{ count: 10 }`. 
```
